src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,22 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment }) => (
+  <>
```

Comment:
You don't need `<></>` here since all the code is inside of `<div className="CommentInfo"></div>`

Reply:
agreed, let's remove fragment